src/Mapping/Annotation/UploadableField.php

```diff
@@ -51,12 +52,26 @@ class UploadableField
      *
      * @throws \InvalidArgumentException
      */
-    public function __construct(array $options)
-    {
-        if (empty($options['mapping'])) {
+    public function __construct(
+        array $options = [],
+        string $mapping = null,
```

Comment:
Why we need to add such arguments?

Reply:
In order to use Named params (https://wiki.php.net/rfc/named_params#:~:text=Named%20arguments%20allow%20passing%20arguments,allows%20skipping%20default%20values%20arbitrarily.) and autocomplete when writing an attribute.

I kept "options" parameters for backward compatibility.